protocol/operation.re

```diff
@@ -52,10 +52,16 @@ module Main_chain = {
 
 module Side_chain = {
   // TODO: I don't like this structure model
+  [@deriving yojson]
+  type timestamp = float;
+
   [@deriving yojson]
   type kind =
     | Transaction({destination: Wallet.t})
-    | Withdraw({owner: Tezos_interop.Address.t});
+    | Withdraw({owner: Tezos_interop.Address.t})
+    | Add_validator(Validators.validator, timestamp)
+    | Remove_validator(Validators.validator, timestamp);
```

Comment:
Also I would say those two should be a single operation as every comment will always apply to both of them.

Reply:
Let's revisit this again at the end.